File: app/src/main/java/com/example/pedidosexpress/views/RepartidorActivity.java
package com.example.pedidosexpress.views;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;
import android.widget.TextView;
import com.example.pedidosexpress.MainActivity;
import com.example.pedidosexpress.R;

public class RepartidorActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.repartidor);

        // DECLARACION DEL ID DE LOS BOTONES, Y LINKS
        Button btnIniciarR = findViewById(R.id.btnIniciarR);
        TextView btnAddCuenta = findViewById(R.id.createAccountLink);
        TextView btnRecuperarPw = findViewById(R.id.forgotPasswordLink);
        ImageView btnback = findViewById(R.id.btnback);

        // METODOS O ACCIONES A REALIZAR
        btnIniciarR.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(RepartidorActivity.this, FeedRepartidor.class);
                startActivity(intent);
            }
        });

        btnAddCuenta.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                // Inicia la actividad AddCuenta.java
                Intent intent = new Intent(RepartidorActivity.this, AddCuenta.class);
                startActivity(intent);
            }
        });

        btnRecuperarPw.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                // Inicia la actividad AddCuenta.java
                Intent intent = new Intent(RepartidorActivity.this, Recuperacion.class);
                startActivity(intent);
            }
        });

        // El botón Back simplemente puede regresar a la actividad pasada.
        btnback.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                onBackPressed(); // Llama al método para regresar a la actividad anterior
            }
        });
    }

    @Override
    public void onBackPressed() {
        super.onBackPressed(); // Esto es importante para mantener el comportamiento predeterminado de ir hacia atrás
        // Si no deseas realizar ninguna acción adicional, no es necesario agregar más código aquí.
    }

}

File: app/src/main/java/com/example/pedidosexpress/views/tienda/RepartidorActivity.kt
package com.example.pedidosexpress.views.tienda

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import android.widget.ImageView
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.example.pedidosexpress.R
import com.example.pedidosexpress.views.recuperacion.Recuperacion
import com.example.pedidosexpress.views.registro.AddCuenta

class RepartidorActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.repartidor)

        // DECLARACION DEL ID DE LOS BOTONES, Y LINKS
        val btnIniciarR = findViewById<Button>(R.id.btnIniciarR)
        val btnAddCuenta = findViewById<TextView>(R.id.createAccountLink)
        val btnRecuperarPw = findViewById<TextView>(R.id.forgotPasswordLink)
        val btnback = findViewById<ImageView>(R.id.btnback)

        // METODOS O ACCIONES A REALIZAR
        btnIniciarR.setOnClickListener {
            val intent = Intent(this@RepartidorActivity, FeedRepartidor::class.java)
            startActivity(intent)
        }
        btnAddCuenta.setOnClickListener { // Inicia la actividad AddCuenta.java
            val intent = Intent(this@RepartidorActivity, AddCuenta::class.java)
            startActivity(intent)
        }
        btnRecuperarPw.setOnClickListener { // Inicia la actividad AddCuenta.java
            val intent = Intent(this@RepartidorActivity, Recuperacion::class.java)
            startActivity(intent)
        }

        // El botón Back simplemente puede regresar a la actividad pasada.
        btnback.setOnClickListener {
            onBackPressed() // Llama al método para regresar a la actividad anterior
        }
    }

}